newdle/client/src/actions.js

```diff
@@ -170,3 +172,19 @@ export function setAnswerActiveDate(date) {
 export function updateNewdle(newdle) {
   return {type: NEWDLE_UPDATED, newdle};
 }
+
+export function fetchBusyTimesForAnswer(participantEmail, dates) {
+  return async dispatch => {
+    dates.forEach(async date => {
```

Comment:
Can't we do it in one go (one request)?

Reply:
i thought about this as well - since this is for a single participant i think fetching it at once would be fine, unless the api has bad performance. OTOH, for a big newdle with many slots, performance with separate requests may be better.

anyway, this is currently using an API not meant for it just to have something to test with. and that particular API is day-based since the creation page only shows one day at a time.

Reply:
I thought about it too but it doesn't seem that heavy to me. We're talking of max. a few days at a time, which may indeed cause problems if the API is slow (as @ThiefMaster suggests), but otherwise pretty normal for a modern web app.

Reply:
Yeah, I'm doing it this way for performance reasons

Reply:
Okay, It seems like everyone is more into one request, so I'll fix that :)

Reply:
I would wait with that until we add an API to get your own availability. It probably won't be a big change here anyway.